# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/DocumentL10n.cpp
# Commit: 85194e5b7a9e
# Full Hash: 85194e5b7a9e6ed9114ee46aef0b5efb819cca9b
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-04 03:56:33
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part9: Update DOMLocalization to match Localization constructor. r=nika,emilio
#   
#   Depends on D113691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113692
# ==============================================================================

diff -r 181487bdf6c0 -r 85194e5b7a9e dom/l10n/DocumentL10n.cpp
--- a/dom/l10n/DocumentL10n.cpp	Tue Aug 03 16:25:13 2021 +0000
+++ b/dom/l10n/DocumentL10n.cpp	Tue Aug 03 16:25:13 2021 +0000
@@ -48,7 +48,7 @@
 }
 
 DocumentL10n::DocumentL10n(Document* aDocument, bool aSync)
-    : DOMLocalization(aDocument->GetScopeObject(), aSync, {}),
+    : DOMLocalization(aDocument->GetScopeObject(), aSync),
       mDocument(aDocument),
       mState(DocumentL10nState::Constructed) {
   mContentSink = do_QueryInterface(aDocument->GetCurrentContentSink());